### ISSUE TITLE
Feature request - Sounder device

### ISSUE THREAD

**Issue by mr-sneezy** [bruhautomation/ESP-MQTT-JSON-Multisensor#59]
Is it possible to add functionality for a buzzer drive from a spare I/O (like D8 perhaps) and control it by MQTT commands. For a door chime beeper from HA or general purpose audible alert from HA ?

**Comment by consolacion** [comment]
Yes I guess it is, but as Ben hasn't been very active lately, I think you'd better put it in yourself, rather than weight for it to be included
